components/channel_members_modal/channel_members_modal.tsx

```diff
@@ -31,7 +31,7 @@ type Props = {
         openModal: (modalData: {
             modalId: string;
             dialogProps: {};
-            dialogType: (props: {}) => React.ReactElement | null;
+            dialogType: (props: any) => React.ReactElement | null;
```

Comment:
Do we not have a type for this? We should try to avoid `any` wherever we can.

Reply:
The issue is that the component being referenced is not typescript so I cant simply import the props for that component into here - is it worth creating the prop types in this file? just to add it as a type here?

Reply:
Ahh, that's unfortunate. No that's alright, we can import them in the future once that component is converted.